client/src/Components/Layouts/SearchPage/SearchPage.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from 'react'
+import Header from '../../Common/Header/Header'
+import Navbar from '../../Common/Navbar/Navbar'
+import SearchIcon from '@material-ui/icons/Search'
+import grey from '@material-ui/core/colors/grey'
+import InputBase from '@material-ui/core/InputBase'
+import axios from 'axios'
+import { Grid } from '@material-ui/core'
+import ArtCard from '../../Common/ArtCard/ArtCard'
+import useStyles from './style'
+
+const SearchPage = () => {
+  const [searchValue, setValue] = useState('')
+  const [art, setArt] = useState(null)
+  const classes = useStyles()
+
+  useEffect(() => {
+    if (art) {
+      return
+    }
+    axios
+      .get('/api/get-art')
+      .then((result) => setArt(result.data))
+      .catch((err) => console.log(err))
+  }, [art])
+
+  const filterSearch = () => {
+    if (!searchValue) {
+      return art
+    }
+    return art.filter(
+      (artwork) =>
+        artwork.titel
+          .toLowerCase()
+          .includes(searchValue.toLowerCase()) ||
+        artwork.category
+          .toLowerCase()
+          .includes(searchValue.toLowerCase()),
+    )
+  }
+
+  return (
+    <div>
+      <Grid container className={classes.root}>
+        <Header pageName="Search" />
+
+        <div className={classes.search}>
+          <div className={classes.searchIcon}>
+            <SearchIcon style={{ color: grey[700] }} />
+          </div>
+          <InputBase
+            placeholder="Search"
+            classes={{
+              root: classes.inputRoot,
+              input: classes.inputInput,
+            }}
+            inputProps={{ 'aria-label': 'search' }}
+            onChange={(e) => {
+              setValue(e.target.value)
+            }}
```

Comment:
This suggestion is potentially a "micro-optimizations" but extracting these to handler expressions means that the runtime doesn't need to make the function each time. Also "point-free" code is awesome. So something like this would be slightly more preferable.

```js
const handleChange = (e) => { setValue(e.target.value) }

...

onChange={handleChange}
```